cmd/docker-app/deploy.go

```diff
@@ -61,13 +65,13 @@ func runDeploy(dockerCli command.Cli, flags *pflag.FlagSet, appname string, opts
 		return err
 	}
 	d := cliopts.ConvertKVStringsToMap(opts.deployEnv)
-	rendered, err := render.Render(app.AppName, opts.deployComposeFiles, opts.deploySettingsFiles, d)
+	rendered, err := render.Render(app, d)
 	if err != nil {
 		return err
 	}
 	stackName := opts.deployStackName
 	if stackName == "" {
-		stackName = internal.AppNameFromDir(app.AppName)
+		stackName = internal.AppNameFromDir(app.Path)
```

Comment:
Would it make sense to have this as a method on `App`?

Reply:
@shin- yeah, I did the *smallest* change possible still, but all the `internal.App*` functions could definitely be attached on the `App` struct :stuck_out_tongue: